README.md

```diff
@@ -21,71 +21,63 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
+## Suported variables
 
-To analyse the project located in `/path/to/project`, execute:
+### Docker arguments and environment variables
 
 ```
-docker run -e SONAR_HOST_URL=http://foo.acme:9000 -it -v "/path/to/project:/usr/src" sonarsource/sonar-scanner-cli
+ARG JDK_VERSION=11
+ARG SONAR_SCANNER_FILE=sonar-scanner-cli-${SONAR_SCANNER_VERSION}-linux.zip
+ARG SONAR_SCANNER_VERSION=4.2.0.1873
+ARG SONAR_SOURCE_KEY=F1182E81C792928921DBCAB4CFCA4A29D26468DE
+
+ENV JDK_VERSION=${JDK_VERSION}
+ENV PATH=$PATH:${SONAR_SCANNER_HOME}/bin
+ENV SONAR_SCANNER_FILE=${SONAR_SCANNER_FILE}
+ENV SONAR_SCANNER_HOME=/opt/sonar-scanner
+ENV SONAR_SCANNER_ID=1001
+ENV SONAR_SCANNER_USER=scanner-cli
+ENV SONAR_SCANNER_VERSION=${SONAR_SCANNER_VERSION}
+ENV SONAR_SOURCE_KEY=${SONAR_SOURCE_KEY}
 ```
 
-To analyse the project in the current directory, execute:
-
-```
-docker run -e SONAR_HOST_URL=http://foo.acme:9000 -it -v "$(pwd):/usr/src" sonarsource/sonar-scanner-cli
-```
-
-## Supported environment variables
-
-### SonarQube host and authentication
-
-* `SONAR_HOST_URL`: URL to the SonarQube instance the scanner should connect to (default=`http://localhost:9000`)
-* `SONAR_TOKEN`: the prefered way to authenticate to SonarQube is to use a token. Use this environment variable to pass it to the scanner
-* `SONAR_LOGIN` and `SONAR_PASSWORD`: alternatively, you can provide the SonarQube username and password for the scanner to use using these two variables
-
 ### Project mounting point
 
 By default, the scanner analyses the project in directory `/usr/src`.
 
-If you can't mount the project into this directory (eg. in Gitlab CI), use `SONAR_PROJECT_BASE_DIR` to specify another location in the container.
-
-```
--e SONAR_PROJECT_BASE_DIR=/build
-```
-
 ### Scanner user home
 
 The scanner downloads data from the SonarQube server it connects to. Retrieving this data can take time and certainly takes bandwidth. For efficiency, the scanner caches this data in the user home (directory named `.sonar`).
 
-When running the scanner with this image, this `.sonar` directory is created in the project's directory (see `SONAR_PROJECT_BASE_DIR`). This implies caching is not happening accross analysis of multiple projects.
-
-Caching is actually shared between projects when running the scanner natively as the `.sonar` is created in the home directory of the current user (eg. `/home/my_user/.sonar`).
-
-You can reproduce this behavior, by adding the following to the `docker run` command:
+When running the scanner with this image, this `.sonar` directory is created in the project's directory. This implies caching is not happening accross analysis of multiple projects.
 
-```
--e SONAR_USER_HOME=/usr/.sonar -v "/home/my_user/.sonar:/usr/.sonar"
-```
+Caching is actually shared between projects when running the scanner natively as the `.sonar` is created in the home directory of the current user.
```

Comment:
This piece of text now confuses me a bit more, is `.sonar` created in the project directory or in the home directory? 
How do I configure it? 
Linking to #27, as this isn't per se part of this PR.

Reply:
Hi @lenntt, 
`.sonar` is created in the project directory when using the Docker container, so yes, that needs to be rewritten.

I don't really know if it is possible to configure.

I suggest that you add the following to any `.gitignore` files.

```
.sonar
.scannerwork
```

Reply:
@lenntt updated the README.